File: New/Hacker Rank/Assignment 1/cpp/Simple Array Sum.cpp
//Simple Array Sum
#include<iostream>
using namespace std;
int simpleArraySum(int ar[],int n)
{
    int sum=0;
    for(int i=0;i<n;i++)
    {
        sum=sum+ar[i];
    }
    return sum;
}
int main()
{
    int n=0,input,results;
    cin>>n;
    int ar[n];
    for(int i=0;i<n;i++)
    {
        cin>>ar[i];
    }
    results=simpleArraySum(ar,n);
    cout<<results;

    return 0;
}

File: New/Hacker Rank/Assignment 1/py/Scripts/Simple Array Sum.py
#Simple Array Sum
def simpleArraySum(ar,n):
    sum=int(0)
    for i in range(n):
        sum=sum+ar[i]
    return sum

n=int(input())
ar=list()
for i in range(n):
    ar.append(int(input()))
results=simpleArraySum(ar,n)
print(results)